srv/salt/ceph/rbd/benchmarks/create_image.sls

```diff
@@ -1,3 +1,11 @@
-create benchmark image:
+create benchmark pool:
   cmd.run:
-    - name: rbd create --size 2048 deepsea_benchmark
+    - name: ceph osd pool create {{ salt['pillar.get']('rbd_benchmark_pool', 'deepsea_rbd_benchmark') }} 256 256
```

Comment:
The pg num needs to be computed.

Reply:
While we're at it, let's compute pg num for all pools DS creates.